AdminUi/src/AdminUi/ClientApp/src/app/services/identity-service/identity.service.ts

```diff
@@ -1,7 +1,7 @@
 import { HttpClient } from "@angular/common/http";
 import { Injectable } from "@angular/core";
 import { NGXLogger } from "ngx-logger";
-import ODataFilterBuilder from "odata-filter-builder";
+import { ODataFilterBuilder } from "odata-filter-builder";
```

Comment:
Can you talk to Daniel A. about this? Because he fixed this in another way. 
And I don't think that your code does actually compile, does it?

Reply:
Sure, I will ask him. But with these changes the app does compile and is running normally on my side.